Sources/AccessibilitySnapshot/Core/Swift/Classes/AccessibilityHierarchyParser.swift

```diff
@@ -601,9 +601,28 @@ private extension NSObject {
                     )
                 )
             }
+
+            let explicitOrderingExceptionList = [
+                "UILayoutContainerView",
+                "UpdateCoalescingCollectionView",
+            ]
+
+            let explicitlyOrdered = accessibilityElements.contains {
+                explicitOrderingExceptionList.contains("\(type(of: $0))") == false
+            }
+            let shouldBeExplicitlyOrdered = accessibilityHierarchyOfElements.contains { node in
```

Comment:
Here is another approach to determining whether a group should be explicitly ordered. I need to think it through a bit more but I think this matches how VoiceOver works since the ordering is determined by the order of the `accessibilityElements` array.